services/user.js

```diff
@@ -65,7 +66,6 @@ class User {
 
     async updateMyUser(params, context) {
         
-        if (!context?.userInfo?.email || !context?.userInfo?.IDP) throw new Error("A user must be logged in to call this API");
```

Comment:
I am not sure why this was removed, this is verifying that the user has logged in before trying to update their user object